sbws/lib/v3bwfile.py

```diff
@@ -43,13 +43,17 @@
 STATS_KEYVALUES = ['number_eligible_relays', 'minimum_number_eligible_relays',
                    'number_consensus_relays', 'percent_eligible_relays',
                    'minimum_percent_eligible_relays']
+# KeyValues to count the number of relays not included in the bandwidth file
+# of for which is not reported a bandwidth value.
```

Comment:
This comment is really hard to understand, and I don't think it matches the latest code.

Did you mean:
```
# KeyValues that count the number of relays that are in the bandwidth file,
# but ignored by Tor when voting, because they do not have a measured bandwidth.
```